mysql-test/suite/innodb/t/percona_log_slow_innodb.test

```diff
@@ -0,0 +1,100 @@
+#
+# Test that InnoDB stats for the extended slow query log are collected.
+#
+--source include/have_innodb.inc
+
+CREATE TABLE t1(a INT AUTO_INCREMENT PRIMARY KEY) ENGINE=InnoDB;
+INSERT INTO t1 VALUES ();
```

Comment:
Don't we need more than 1 record here as in 5.5?

Reply:
5.5 needs not more than one record, but more than one index page, otherwise, on table opening it will be read for stats, putting the whole table in the buffer pool. 5.6 does not need this due to persistent stats. Or is it a different issue you had in mind?

Reply:
Yes, exactly the issue you described. Understood.